### PR TITLE
English routes #67

### PR DESCRIPTION
**Pull request by timozattol**

### CODE REVIEW

File: database/seeds/FAQSeeder.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ public function run() {
 
 		$question_answers_en = [
 			'I want to offer a job to a student, how to proceed?'	=>
-				'You simply have to create an ad following <a href="nouveau-job">
+				'You simply have to create an ad following <a href="new-job">
```
Comment (zifeo) [inline database/seeds/FAQSeeder.php]: Using `' + action('AdController@created') + ' ` could provide a better solution for long-term, same below.

**Comment by zifeo** [comment]
:+1:

File: database/seeds/FAQSeeder.php (hunk 1 above)
Comment (timozattol) [inline database/seeds/FAQSeeder.php]: Right! I'll update that

**Comment by zifeo** [comment]
LGTM

**Pull request merged**